### PR TITLE
:heavy_plus_sign: Added PG21 paper

### PR DESCRIPTION
**Pull request by reihos**
Added the 2021 Powders and Grain paper to the publications.

### CODE REVIEW

**Review by kks32 — CHANGES_REQUESTED** [review]
Could you please add your Abstract below?

**Comment by reihos** [comment]
Like this?

File: _posts/publications/2021-01-04-multiphase-swrc.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+layout: publication
+categories: 
+  - publications
+tags:
+  - conference
+  - multiphase-lbm
+  - hpc
+preprint: https://arxiv.org/abs/2102.08420
+abstract:
+The soil water retention curve (SWRC) is the most commonly used relationship in the study of unsaturated soil. In this paper, the effect of porosity on the SWRC is investigated by numerically modeling unsaturated soil using the Shan-Chen multiphase Lattice Boltzmann Method. The shape of simulated SWRCs are compared against that predicted by the van Genuchten model, demonstrating a good fit except at low degrees of saturation. The simulated SWRCs show an increase in the air-entry value as porosity decreases.
```
Comment (kks32) [inline _posts/publications/2021-01-04-multiphase-swrc.md]:
```
---

## Abstract

The soil water retention curve (SWRC) is the most commonly used relationship in the study of unsaturated soil. In this paper, the effect of porosity on the SWRC is investigated by numerically modeling unsaturated soil using the Shan-Chen multiphase Lattice Boltzmann Method. The shape of simulated SWRCs are compared against that predicted by the van Genuchten model, demonstrating a good fit except at low degrees of saturation. The simulated SWRCs show an increase in the air-entry value as porosity decreases.

```

Comment (kks32) [reply _posts/publications/2021-01-04-multiphase-swrc.md]: below the `---`

**Review by kks32 — CHANGES_REQUESTED** [review]
Add the abstract below the line

**Review by kks32 — APPROVED** [review]

**Pull request merged**